src/app_service/service/wallet_account/utils.nim

```diff
@@ -102,3 +102,12 @@ proc hasPairedDevices(): bool =
     return response.result.getBool
   except Exception as e:
     error "error: ", errDesription=e.msg
+
+proc balanceInfoToTable(jsonNode: JsonNode): Table[string, UInt256] =
+  for pair in jsonNode.pairs():
+    for pair2 in pair.val.pairs():
+      for pair3 in pair2.val.pairs():
+        let amount = fromHex(UInt256, pair3.val.getStr)
+        if amount != stint.u256(0):
+          result[pair2.key.toUpper] = amount
+        break
```

Comment:
@MishkaRogachev shouldn't this `break` need to be under `then` block of the `if` condition?

Reply:
that's code from temporary commit i forget to exclude. Fixed